src/entities/Surface.js

```diff
@@ -35,10 +48,12 @@ export class Surface extends Entity {
      *
      */
     connected() {
-        this.windowVerticalScale = this.height / 3;
-        this.windowHorizontalScale = this.width / 3;
+        // this.clipping = new ClippingGeometry(new THREE.BoxGeometry(this.width, this.height, 0.3));
+        this.windowVerticalScale = this.height; /// 3;
+        this.windowHorizontalScale = this.width; // / 3;
```

Comment:
question for understanding: (i see this was moved to line 67) is the divide by 3 just a choice or is there a reason we're using 3 specifically?

Reply:
> question for understanding: (i see this was moved to line 67) is the divide by 3 just a choice or is there a reason we're using 3 specifically?

This is a choice atm, the window is really big if we stick to 1:1 scale. I'd like to refactor the anchoring/surface system overall. so we can parameterize this in the future?

Reply:
sounds good - then in line 67, either add a comment or can we separate the divide by 3 into a constant first and then divide - better visual separation for future parameterization - just dont want it to end up as a 'secret special number'

Reply:
Made it a global.